Documentation/Signoffs/Power_System_Receiver.md

```diff
@@ -1,37 +1,148 @@
+| Total     |                |                |  0.018          |  0.01483         |  0.465          |  0.0639         |  0.6225     |   2.0845      |  <br />
+
+
+Table 2: Power Consumption Calculations<br />
+
+<br />
+
+## Arduino Nano ESP32
+The Arduino® Nano ESP32 can be powered with 5 V through a USB-C connector or an input voltage of 6-21 volts through the VIN pin. For this design, the team will be using the 5 V USB-C port, because this gives the microcontroller additional functionality that may be utilized in the receiver system's design by allowing the usage of the VBUS 5 V output. This will also allow the output terminals to be free power the ESP8266 WiFi module if required. Since the ESP32's output voltage is 3.3 V, it can also be used to power the ESP8266. The best design decision in this case will likely be determined during the testing period, and having two different options for connection is beneficial for design flexibility. The decision to use this component was decided by the receiver system design team.  <br />
+## ESP8266 WiFi Module
+The ESP8266 can be powered through VIN pin at 3.3 V. This component allows the receiver system to connect to WiFi and communicate with the database system. The decision to use this component was decided by the receiver system design team.<br />
+## Waveshare Solar Power Management Module
+The Waveshare Solar Power Management Module is a key component for the subsystem's design, since all the other electrical components interface with it. The module uses a 14500 2500 mAh 3.7 V Li-Ion battery as the source. That battery can be recharged by solar panels ranging from 6 - 24 V and a Micro-USB port, which will be helpful for testing purposes. The module also has flexibility for output voltages with a USB-A output port, 5 V output terminal, and a 3.3 V terminal. This module's flexibility allows the design to save money, since it is capable of completing multiple functions alongside its inexpensive price. The Waveshare module also has a variety of safety features such as over-charge, over-discharge, reverse protection, over heat, and over current protections that will prevent accidents and damage to the system. It also utilizes Maximum Power Point Tracking to maximize the efficiency of the solar panel. This product was chose by the design team because of the functionality provided, because helps reduce design time and simplifies the design by serving many purposes to the system with a single component. It was also important to find a cheap product, because of the need to power nine receiver systems in different locations across campus. One downside of using this module is the complexity of the electronics makes it very hard to accurately depict its functionality in a simulation. To combat this, the design team will use values that are slightly higher than expected in our calculations to account for losses or unforseen circumstances.
```

Comment:
Suggestion: the word "cheap" is commonly associated with low-performance or low-quality so it might be good to change to a different word such as "inexpensive".

Reply:
Wording changed